src/udp_wrap.h

```diff
@@ -98,6 +99,10 @@ class UDPWrapBase {
   // Stores the sockaddr for the local socket in `name`.
   virtual int GetSockName(sockaddr* name, int* namelen) = 0;
 
+  virtual SocketAddress* GetPeerName(SocketAddress* addr = nullptr) = 0;
+
+  virtual SocketAddress* GetSockName(SocketAddress* addr = nullptr) = 0;
```

Comment:
This should be fine, but in the long run either the `SocketAddress*` variants or the `sockaddr*` variants should become non-virtual and implemented in terms of the other, I guess.

Also, could these maybe just return a `SocketAddress` instead of taking a pointer argument that doubles as the return value?

Reply:
+1 to the first part... will look at that in a future PR.

For the second part, I'll look at that but the current use of this in node_quic_socket.cc updates a member variable of `QuicSocket`...

```c++
udp_->GetSockName(&local_address_);
```

The return value isn't actually used. I set it up this way just to give some flexibility that we may not actually need in the long run. That is, we could alternatively just drop the return value.